🌐 Add German translation for `docs/de/docs/tutorial/dependencies/index.md`

nilslindemann opened the pull request:
← `tutorial/body-updates.md` (#10396)
→ `tutorial/dependencies/classes-as-dependencies.md` (#10407)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

Commit:
🌐 Add German translation for `docs/de/docs/tutorial/dependencies/index.md`

* Full translation
* In sync with current docs/en/docs/tutorial/dependencies/index.md
* Google Translate used
* Manually checked
* Checked for errors using LanguageTool

tiangolo commented:
📝 Docs preview for commit 2121d39200a8b9326cb0121085824c519b36a938 at: https://c3641dbf.fastapitiangolo.pages.dev

Commit:
"Dependency" -> "Abhängigkeit"

tiangolo commented:
📝 Docs preview for commit 470ffc500c41b9e06f3c9ce83e412cfb00b2bf85 at: https://62171c27.fastapitiangolo.pages.dev

Commit:
Give tooltip for first usage of "Dependable" and "Dependant"

tiangolo commented:
📝 Docs preview for commit 054d0c7a94ec1d02a87ddc980346088c5eb7a019 at: https://d13edf84.fastapitiangolo.pages.dev

Commit:
Translate link pointing to async.md

That document is now translated, see https://github.com/tiangolo/fastapi/pull/10449

tiangolo commented:
📝 Docs preview for commit 43a423283ea4b649f52edff10f1fa9b409c7112b at: https://37fbe2ab.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/dependencies/index.md

Commit:
Sync with #10442

tiangolo commented:
📝 Docs preview for commit c70e863c4fb43c96c4a0ec07d70125f5cc307bfc at: https://1ba2aae3.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/dependencies/index.md

tiangolo commented:
📝 Docs preview for commit b21d1d494edd833de93984e83bd0a826b90b6c25 at: https://4f966c74.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/dependencies/index.md

tiangolo commented:
📝 Docs preview for commit e8136bb924be7ebd73f50ee0ef6003e130ed0f36 at: https://910f5b3e.fastapitiangolo.pages.dev

Commit:
Translate link pointing to `deployment/versions.md`

That document is now translated, see #10491

tiangolo commented:
📝 Docs preview for commit 4311086df0261753a2b50fb96026f2b261c213bf at: https://b3093b3b.fastapitiangolo.pages.dev

Commit:
"Tip" -> "Tipp"

tiangolo commented:
📝 Docs preview for commit 9504babcc2b5ddae9f1b3892d0c654a8b39b4d7f at: https://36cadcb8.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/dependencies/index.md

tiangolo commented:
📝 Docs preview for commit 9fbfdfdc041000f39caf2f37315e83fa1c46e3b3 at: https://fe2725ca.fastapitiangolo.pages.dev

Commit:
Use typographic quotes

tiangolo commented:
📝 Docs preview for commit 8fbe4ce71307d633ec76a94f31330ac26a685cdc at: https://e8dc49a5.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/dependencies/index.md

tiangolo commented:
📝 Docs preview for commit 7f67730a50e6bddad265244c118053821215778c at: https://fc3f5058.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/dependencies/index.md

tiangolo commented:
📝 Docs preview for commit d7f4cd8c207b4c5604d4a8a082c4a19b0b6e3b20 at: https://1520ae0b.fastapitiangolo.pages.dev

Commit:
"Standard " -> "Standard-"

tiangolo commented:
📝 Docs preview for commit 85921957912ac9e155fe07173168ca2b72207184 at: https://375eaecc.fastapitiangolo.pages.dev

Commit:
"das gleiche" vs "dasselbe"

tiangolo commented:
📝 Docs preview for commit c3f4e5d1d1ba4bc756b997ffe816d16189a5f359 at: https://f33d9ced.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/dependencies/index.md

tiangolo commented:
📝 Docs preview for commit daba23b80ee7730fd14cd34ef369b1658bd4de4e at: https://798881d7.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/dependencies/index.md

tiangolo commented:
📝 Docs preview for commit 6e65f71794d53ef3bb70c2328a57662ad5cb756c at: https://4bc2bed6.fastapitiangolo.pages.dev

Commit:
"Web-Framework" -> "Webframework"

tiangolo commented:
📝 Docs preview for commit 9d5ed0eaa7a22001b724e3bb585d0290ee1258b3 at: https://6db08231.fastapitiangolo.pages.dev

Commit:
Remove "-"

tiangolo commented:
📝 Docs preview for commit fe33c7bfa4173f621177eb42c643142d74e14dcb at: https://cece3e2f.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/dependencies/index.md

tiangolo commented:
📝 Docs preview for commit ba1f8b89b4dc93fd0ebfd4869edb57a8b3ac8e45 at: https://fcd4d958.fastapitiangolo.pages.dev

tiangolo commented:
Awesome @nilslindemann! :rocket: 

As you are the champion of German translations and you have done so many, and with so much attention to detail, I'm gonna go ahead and merge this. :tada: 

If others see things that could be improved later, we can check that in new PRs. :nerd_face:
